packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade/upgrade-task.class.js

```diff
@@ -0,0 +1,61 @@
+import 'moment';
+
+export default class UpgradeTask {
+  constructor({
+    comment,
+    doneDate,
+    function: type,
+    lastUpdate,
+    needSchedule,
+    note,
+    plannedIntervention,
+    startDate,
+    status,
+    taskId,
+    ticketReference,
+  }) {
+    Object.assign(this, {
+      comment,
+      doneDate,
+      lastUpdate,
+      needSchedule,
+      note,
+      plannedIntervention,
+      startDate,
+      status,
+      taskId,
+      ticketReference,
+      type,
+    });
+  }
+
+  get components() {
+    return this.note
+      ? this.note.split(',').map((component) => component.trim())
+      : [];
+  }
+
+  get scheduledTimeSlot() {
+    return this.plannedIntervention
+      ? moment(this.plannedIntervention.wantedStartDate).format(
+          'YYYY/MM/DD h:mm A',
```

Comment:
can you format date based on locale? it will be confusing otherwise for some customers.

Reply:
This has been handled now